python/pyspark/sql/catalog.py

```diff
@@ -50,7 +50,7 @@ def currentDatabase(self):
     @since(2.0)
     def setCurrentDatabase(self, dbName):
         """Sets the current default database in this session."""
-        return self._jcatalog.setCurrentDatabase(dbName)
+        self._jcatalog.setCurrentDatabase(dbName)
```

Comment:
Shall we mention `setCurrentDatabase` together?

Reply:
if I am not mistaken, previously it return `None` too because the JVM returns `null`:

```python
>>> print(spark.catalog.setCurrentDatabase("default"))
None
```

so it should be fine.

Reply:
Let me merge this in but please let me know if there's something I missed.

Reply:
Ah, got it. It makes sense.